config/forms/2021_2022.json

```diff
@@ -32,7 +32,8 @@
                   "type": "numeric",
                   "min": 0,
                   "max": 150,
-                  "step": 1
+                  "step": 1,
+                  "regex_validation" : "^[1-9][0-9]?$|^100$"
```

Comment:
Given that I think for some validations the logic is likely to be quite complex/not suitable for a regex, I think I'd be inclined to not put it into the JSON for now, or if you do, just put in a method name, where that method contains the validation to be run?

Might be prematurely optimising here though so feel free to ignore that.

Reply:
My hope was to keep all the bits of the form that are subject to change in the same domain. However, in the interest of keeping it as simple as possible I will leave it in it's respective validation method.